Engine/Code/Engine/Core/Time.cpp

```diff
@@ -44,9 +44,4 @@ float Time::GetTimeDelta()
 		d = m_FPS;
 
 	return 1.0f / d;
-}
-
```

Comment:
Again time should not have a concept of seed at all
@kevin30061994 correct me if I am wrong here